<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>950</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Telcorain</string>
  </property>
  <property name="locale">
   <locale language="English" country="World"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabControl">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/adjust.png</normaloff>icons/adjust.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Control Panel</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupCalcType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Calculation Type</string>
          </property>
          <widget class="QRadioButton" name="radioRealtime">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>30</y>
             <width>69</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Realtime</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioTimeint">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>89</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Time Interval</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinTimestep">
           <property name="geometry">
            <rect>
             <x>330</x>
             <y>29</y>
             <width>101</width>
             <height>22</height>
            </rect>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> min</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1440</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>20</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Timestep:</string>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="dateTimeStart">
           <property name="geometry">
            <rect>
             <x>500</x>
             <y>29</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>12</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2022</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="time">
            <time>
             <hour>12</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="maximumTime">
            <time>
             <hour>19</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>20</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="dateTimeStop">
           <property name="geometry">
            <rect>
             <x>670</x>
             <y>29</y>
             <width>111</width>
             <height>22</height>
            </rect>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>12</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2022</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="time">
            <time>
             <hour>12</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="maximumTime">
            <time>
             <hour>19</hour>
             <minute>59</minute>
             <second>59</second>
            </time>
           </property>
           <property name="minimumTime">
            <time>
             <hour>20</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy HH:mm</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::UTC</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>20</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>630</x>
             <y>20</y>
             <width>61</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>Stop:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>800</x>
             <y>20</y>
             <width>151</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <italic>true</italic>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>All times are in UTC format!</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupLinkset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>185</height>
           </size>
          </property>
          <property name="title">
           <string>Link Set</string>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>241</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Links list:</string>
           </property>
          </widget>
          <widget class="QListView" name="listLists">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>52</y>
             <width>241</width>
             <height>111</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="buttLstNew">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>50</y>
             <width>91</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>New...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttLstEdit">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>80</y>
             <width>91</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Edit...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttLstCopy">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>110</y>
             <width>91</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttLstDel">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>140</y>
             <width>91</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QListView" name="listLinks">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>52</y>
             <width>551</width>
             <height>111</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>30</y>
             <width>241</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Microwave links included in selected list:</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupParams">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>130</height>
           </size>
          </property>
          <property name="title">
           <string>Parameters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupWaa">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>130</height>
           </size>
          </property>
          <property name="title">
           <string>WAA Compensation</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="buttLoad">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttSave">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttDef">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Defaults</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Show results in viewer:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShowIn1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShowIn2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShowIn3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShowIn4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioShowIn5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttStart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string> Start Calculation</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/play.png</normaloff>icons/play.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttAbort">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string> Abort Calculation</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>icons/cancel.png</normaloff>icons/cancel.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabInspector">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/inspect.png</normaloff>icons/inspect.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Link Inspector</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabLog">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/output.png</normaloff>icons/output.png</iconset>
       </attribute>
       <attribute name="title">
        <string>System Log</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults1">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/explore.png</normaloff>icons/explore.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Results Viewer 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults2">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/explore.png</normaloff>icons/explore.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Results Viewer 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults3">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/explore.png</normaloff>icons/explore.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Results Viewer 3</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults4">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/explore.png</normaloff>icons/explore.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Results Viewer 4</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults5">
       <attribute name="icon">
        <iconset>
         <normaloff>icons/explore.png</normaloff>icons/explore.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Results Viewer 5</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="iconText">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Terminate the program and exit.</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioRealtime</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateTimeStart</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioRealtime</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateTimeStop</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTimeint</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateTimeStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioTimeint</sender>
   <signal>clicked(bool)</signal>
   <receiver>dateTimeStop</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
